data/filetypes.rust

```diff
@@ -62,11 +62,22 @@ context_action_cmd=
 # 0 is spaces, 1 is tabs, 2 is tab & spaces
 #type=1
 
-[build_settings]
-# %f will be replaced by the complete filename
-# %e will be replaced by the filename without extension
-# (use only one of it at one time)
-compiler=rustc "%f"
-linker=rustc -o "%e" "%f"
-run_cmd="./%e"
-
+[build-menu]
+FT_00_LB=Compile
+FT_00_CM=rustc "%f"
+FT_00_WD=
+FT_01_LB=Compile
```

Comment:
It would appear that someone has assumed that the rust compiler works like C.  If this isn't needed remove it, but certainly do not name both commands "Compile".